### DISCUSSION TITLE
Carousel Type Showcase is not applied

### DISCUSSION THREAD

**Discussion by calvin123** [ImaginativeShohag/Why-Not-Image-Carousel#54] [Q&A]
Dear all,

I am just getting started with Android development, but I was happy to find Why-Not-Image-Carousel.
Sadly I do struggle with setting the widget to "Showcase" mode.

I tried to do it in the xml with `app:carouselType="SHOWCASE"` as well as trying 
`ImageCarousel carousel = findViewById(R.id.carousel);`
`carousel.setCarouselType(CarouselType.SHOWCASE);`
in my init method in Java.

Are there any more constraints that need to be met?

Thanks for reading
cal

**Comment by ImaginativeShohag** [comment]
Please clone this repo and check out the sample in the [sample](https://github.com/ImaginativeShohag/Why-Not-Image-Carousel/tree/master/sample) folder. Ask me again if you don't find your answer there.
